src/elements/Progressbar/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames/bind';
+import style from './style.scss';
+import numberInRange from '../../utils/numberInRange';
+
+const classNames = cx.bind(style);
+const colorMap = {
+  success: 'color-success',
+  warning: 'color-warning',
+  error: 'color-error',
+};
+
+const Progressbar = ({ percentage = 0, showPercentageNum, className, color, ...props }) => {
+  const progressBarWidth = percentage > 100 ? 100 : percentage;
+  const percentageLabel = showPercentageNum ? `${percentage}%` : null;
+  const progressBarColor = colorMap[color] || 'color-success';
```

Comment:
we already set the default value from `Progressbar.defaultProps`. Also, why not `primary` color for default progressbar? And we don't need to mapping since it is just `color-` + property value. I think we don't need `color-` prefix for class. If we want to have specifically progressbar, then we can do that in jsx itself.